status-im/status-react
# fix: remove non operable accounts in the selection of sharing addresses with the community

fixes #20624

### Summary

This PR removes the non-operable accounts in the `Addresses for permission` and `Airdrop address` selections while joining a community.

https://github.com/status-im/status-mobile/assets/19339952/ed9ed655-bb38-4008-a738-f82b38367660

### Testing notes

A quick check across the app to verify the non-operable accounts are not shown is appreciated.

### Platforms

- Android
- iOS

### Steps to test

- Open Status
- Login with a profile with a missing key pair
- Navigate to a community that the user does not join
- Tap on request to join
- Verify the non-operable accounts are not shown to the user for selection in `Addresses for permission` and `Airdrop address`

status: ready
